<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChooseCommonArea</class>
 <widget class="QDialog" name="ChooseCommonArea">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>147</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../icons/resources.qrc">
    <normaloff>:/png/marca-m.png</normaloff>:/png/marca-m.png</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>531</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string> Escolha o Local</string>
   </property>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>491</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonReservar">
   <property name="geometry">
    <rect>
     <x>422</x>
     <y>106</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Reservar</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancelar">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>107</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>comboBox</tabstop>
  <tabstop>pushButtonCancelar</tabstop>
  <tabstop>pushButtonReservar</tabstop>
 </tabstops>
 <resources>
  <include location="../../icons/resources.qrc"/>
  <include location="../../icons/resources.qrc"/>
  <include location="../../icons/resources.qrc"/>
 </resources>
 <connections/>
</ui>
